autopts/ptsprojects/zephyr/cap.py

```diff
@@ -173,12 +173,17 @@ def set_addr(addr):
     ]
 
     general_conditions = [
-        TestFunc(announcements, adv_data, rsp_data, False),
+        TestFunc(announcements, adv_data, rsp_data, False, SINK_CONTEXTS, SOURCE_CONTEXTS),
         TestFunc(btp.gap_adv_ind_on, ad=adv_data, sd=rsp_data),
     ]
 
     targeted_conditions = [
-        TestFunc(announcements, adv_data, rsp_data, True),
+        TestFunc(announcements, adv_data, rsp_data, True, SINK_CONTEXTS, SOURCE_CONTEXTS),
+        TestFunc(btp.gap_adv_ind_on, ad=adv_data, sd=rsp_data),
+    ]
+
+    idle_targeted_conditions = [
+        TestFunc(announcements, adv_data, rsp_data, True, 0, 0),
```

Comment:
Did you consider the approach of updating the advertising data in `hdl_wid_421` instead of setting it as a pre-condition?

Reply:
Yes, the problem is that the wid is platform agnostic, whereas the format of the ad probably isn't. I ran into that issue when I tried to import the `announcement` function from `ptsproject.zephyr.cap`

Reply:
The advertising data should be the same as what BT SIG defines, so should be platform agnostic. 

@sjanc may have some input here